<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTrackInfo</class>
 <widget class="QDialog" name="DlgTrackInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Track Editor</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lblSong">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Song:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="toolbox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pgTrack">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>360</width>
        <height>351</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Track Information (Click to Expand)</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="txtTrackName"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Artist:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="3">
        <widget class="QLineEdit" name="txtArtist"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Filename:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="txtFilepath">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Play Time:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="txtDuration">
         <property name="text">
          <string>3:00</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="Type">
         <property name="text">
          <string>File Type:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="txtType">
         <property name="text">
          <string>77</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Comments:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="3">
        <widget class="QTextEdit" name="txtComment"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pgCue">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>375</width>
        <height>321</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Track Cuepoints</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="cueTable">
         <column>
          <property name="text">
           <string>Cue Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Hotcue</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Label</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnCueActivate">
           <property name="text">
            <string>Activate Cue</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCueDelete">
           <property name="text">
            <string>Delete Cue</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnPrev">
       <property name="text">
        <string>&lt;&lt; &amp;Prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string>Alt+O</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNext">
       <property name="text">
        <string>&amp;Next &gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
